docs/source/utils.md

```diff
@@ -72,6 +72,16 @@ INFO [osp.core.ontology.parser]: Loaded 205 ontology triples in total
 INFO [osp.core.ontology.installation]: Uninstallation successful
 ```
 
+### Ontology installation folder
+
+The installed ontologies are stored in the directory `~/.osp-ontologies` by 
+default. On Windows, `~` usually refers to the path 
+`C:\Users\username`.
+
+The installation directory can be changed by setting the
+environment variable `OSP_ONTOLOGIES_DIR`. Such action would move it to 
+`OSP_ONTOLOGIES_DIR/.osp-ontologies`.
```

Comment:
Also here: `$OSP_ONTOLOGIES_DIR/.osp-ontologies`

Reply:
Here I would agree to use it, since we are referring to the value of the variable and not the variable itself.